rrfs-test/scripts/setup_experiment.sh

```diff
@@ -98,8 +98,8 @@ elif [[ $DYCORE == "MPAS" ]]; then
   mkdir -p data; cd data
   mkdir -p bumploc bkg obs ens
   ln -snf ${YOUR_PATH_TO_RDASAPP}/fix/bumploc/* bumploc/.
-  ln -snf ${YOUR_PATH_TO_RDASAPP}/fix/expr_data/${TEST_DATA}/bkg/restart.2024-05-27_00.00.00.nc .
-  ln -snf ${YOUR_PATH_TO_RDASAPP}/fix/expr_data/${TEST_DATA}/bkg/restart.2024-05-27_00.00.00.nc static.nc
+  ln -snf ${YOUR_PATH_TO_RDASAPP}/fix/expr_data/${TEST_DATA}/bkg/mpasout.2024-05-27_00.00.00.nc .
+  ln -snf ${YOUR_PATH_TO_RDASAPP}/fix/expr_data/${TEST_DATA}/invariant.nc invariant.nc
```

Comment:
What does `invariant` here refer to? Is it different from an `mpasout` file?

Reply:
The new MPAS-MODEL and MPAS-JEDI now split the model outputs into two parts: invariant and varaint.
The invariant part refers to grids, terrain data, etc. It is the previous `static.nc` but with a vertical grid.
The variant part refers to `mpasout`, `diag` or any other customized streams where their contents varies in different cycles.
This design is to facilitate cycling data assimilation so that we don't need to carry giant files in each data assimilation cycle.

`invariant` is now a keyword in MPAS-MODEL.